core/types/bundle_gasless.go

```diff
@@ -0,0 +1,21 @@
+package types
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+)
+
+type SimulateGaslessBundleArgs struct {
+	Txs []hexutil.Bytes `json:"txs"`
+}
+
+type GaslessTx struct {
```

Comment:
Name `GaslessTx` does not match its meaning, what about "TxSimResult"?

Reply:
changed to `GaslessTxSimResult`